README.md

```diff
@@ -61,3 +61,12 @@ Then run :
     2020-08-24 15:22:20,820 root - INFO - Inserted row 1 
     ...
 
+
+### Flask REST Front End
+
+Create the flask app from the https://github.com/data2health/nlp-sandbox-schemas CodeBase
+
+    cd ~project/nlp-sandbox-schemas
```

Comment:
Replace with something like:

1. Download OpenAPI spec from XXX (we will replace with the url later)
2. Install openapi-generator...
3. Generate initial server

Reply:
Maybe add a section about how to update to server implementation is a new version of the OpenAPI spec is available.

Reply:
Can you also add a section that describes how to start the stack with `docker-compose`?

Reply:
I've been adding doc to the ~/flask-app/README.md Should I move those to the top level README.md ?

Reply:
This look good for now. Later we will need to section: 1) how to deploy the stack using docker-compose, and 2) how to start the server manually (current section) for development purpose.